example/client_authorization_policys.tf

```diff
@@ -0,0 +1,174 @@
+resource keycloak_realm test {
```

Comment:
I think most of these examples are going to conflict with the pre-existing `main.tf` file in the `example` folder.  Could you rename the resources in this file so they don't conflict with `main.tf`?  It's okay to hardcode any strings you want here, these files really just serve as an example of how to use the provider.

Reply:
You are also free to reuse the resources that have already been created in `main.tf`, meaning that you can just leave them in that file and use variable interpolation to depend on them (like you are on line 12).